lib/asset_cloud/asset.rb

```diff
@@ -89,6 +89,10 @@ def value_hash
       metadata.value_hash
     end
 
+    def checksum
```

Comment:
I'd normally use `delegate` for this, but I've followed the existing style for this file.